pipelines/ceus/tasks/fcz_correlation.py

```diff
@@ -51,7 +54,16 @@ def _task(self):
         correlation_matrix = pd.DataFrame(0, index=self.projection_locations, columns=self.fcz_names)
 
         for base in self.fcz_names:
+
+            if base in self.excluded_locations['base']:
```

Comment:
I want excluded_locations to be a user adjustable list separate from fcz list.

Reply:
It can still be a user adjustable list, however, we can leverage things python already has built-in to give us things that are unique to one list. Meaning, that exist in list 1 AND NOT in list 2. 

Two other ways to ask this question is, can python lists do left outer joins? Or can python more easily give us the difference between sets?

Here's an example (syntax is based off of python 2.x):
```

lst_1 = [1,2,3,4,5,6,7,8]
lst_2 = [5,6,7,8,9,11,12,13]
new_lst = []
for i in lst_1:
    if i in lst_2:
        print(i, 'exists in lst two')
    else:
        print(i, 'does not exist in lst two')
        new_lst.append(i)

print(new_lst)  # [1, 2, 3, 4]

new_lst_2 = set([(v if v not in lst_2 else None) for v in lst_1])
print(new_lst_2)  # still need to remove None -  set([1, 2, 3, 4, None])

set_1 = set(lst_1)
set_2 = set(lst_2)
new_set = set_1.difference(set_2)

print(new_set)  # set([1, 2, 3, 4])
```

I guess my suggestion is that by using the builtin set difference op, you can simplify a lot of the logic in this loop.

Reply:
Here's the 3.7 doc: https://docs.python.org/3.7/library/stdtypes.html#frozenset.difference

However, the 2.7 docs are easier to digest oddly enough: https://docs.python.org/2/library/sets.html#set-objects